addons/base_vat/models/res_partner.py

```diff
@@ -222,7 +222,14 @@ def _build_vat_error_message(self, country_code, wrong_vat, record_label):
         else:
             company = self.env.company
 
-        vat_label = company.country_id.vat_label if company.country_id else "Tax ID"
+        country = self.env['res.country'].search([('code', '=ilike', country_code)], limit=1)
```

Comment:
```suggestion
        partner_country = self.env['res.country'].search([('code', '=ilike', country_code)], limit=1)
```

Reply:
@jco-odoo country_code comes in lowercase `in` and in DB it's in uppercase `IN` so use `=like`

Reply:
we could remove all the `.lower()` from the calling arguments but then we need it when lookup the dict, so either way is bad